monitoring/dns.go

```diff
@@ -29,19 +29,54 @@ import (
 type DNSChecker struct {
 	// QuestionA is a slice of questions to ask for A (Host) records
 	QuestionA []string
+	// QuestionNS is a slice of questions to ask for NS (Nameserver) records
```

Comment:
I have an off-topic question about how these changes ended up here?

Reply:
@eldios Yeah, actually I'm kinda curious about it too. Seems unrelated to the ping check?